docker-compose-services/mysql/README.md

```diff
@@ -0,0 +1,40 @@
+For the vast majority of use cases where MySQL is needed in a project, MariaDB is the best choice. MariaDB is faster and more feature-packed in many scenarios. However, sometimes a project really demands plain old MySQL.
+This page documents how to create a separate MySQL container to use with ddev. It does not
+replace the MariaDB or PHPMyAdmin containers. It also demonstrates how to connect Drupal to that container and database.
+
+## ddev setup
+1. Add [docker-compose.mysql.yaml](docker-compose.mysql.yaml) to your  project's `.ddev` directory.
+2. Start or restart your project with `ddev start` or `ddev restart`.
+3. You should see your container start up. You could look for it with `docker ps -a | grep mysql`. The container will be named ddev-[your project name]-mysql.
```

Comment:
4. Use project type 'php' instead of letting ddev adjust your settings.php/settings.ddev.php or equivalent. `ddev config --project-type=php` (or edit .ddev/config.yaml and set `type: php`)